javascript/ql/test/query-tests/Security/CWE-116/IncompleteSanitization/IncompleteSanitization.expected

```diff
@@ -29,12 +29,9 @@
 | tst.js:149:2:149:24 | x.repla ... replace | This replaces only the first occurrence of "\\n". |
 | tst.js:193:9:193:17 | s.replace | This replaces only the first occurrence of /'/. |
 | tst.js:202:10:202:18 | p.replace | This replaces only the first occurrence of "/../". |
-| tst.js:337:2:337:12 | s().replace | This replaces only the first occurrence of new Reg ... nown()). |
```

Comment:
Line 337 still says `// NOT OK`, which doesn't fit with this change?

Reply:
This one was suppose to be caught by [IncompleteBlacklistSanitizer](https://github.com/Napalys/codeql/blob/fd773603e60254653a48a910720ab8082c730cd5/javascript/ql/test/query-tests/Security/CWE-116/IncompleteSanitization/IncompleteBlacklistSanitizer.expected#L73) and not by `IncompleteSanitization` and it was caught as a side effect by switching from `RegExpLiteralNode` -> `RegExpCreationNode`